java/query/src/test/java/sleeper/query/lambda/SqsQueryProcessorLambdaIT.java

```diff
@@ -244,10 +247,11 @@ public void shouldSetStatusOfQueryAndSubQueriesToCOMPLETEDWhenAllSubQueriesHaveF
                 .queryId("abc")
                 .regions(List.of(new Region(List.of(range1, range2, range3))))
                 .build();
+
         processQuery(query);
 
         // When
-        processQueriesFromQueue(4);
+        processLeafPartitionQuery(6);
```

Comment:
This can still be 4 messages rather than 6.

Reply:
Updated to 4